backend/clubs/models.py

```diff
@@ -762,6 +762,7 @@ class ClubVisit(models.Model):
 
     person = models.ForeignKey(get_user_model(), on_delete=models.CASCADE)
     club = models.ForeignKey(Club, on_delete=models.CASCADE)
+    ip = models.CharField(max_length=255, blank=True, null=True)
```

Comment:
Instead of a `CharField`, there is a [GenericIPAddressField](https://docs.djangoproject.com/en/3.1/ref/models/fields/#genericipaddressfield) that you can use.

Reply:
Thanks!